packages/dialect-react-ui/components/DialectProvider/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from 'react';
+import * as anchor from '@project-serum/anchor';
+import {
+  connected,
+  useApi,
+  WalletType,
+  ApiProvider,
+  DialectProvider as DialectProviderInternal,
```

Comment:
What's the motivation for two providers, one internal?

Reply:
It's to avoid naming collision. We have `ApiProvider` and `DialectProvider` in `@dialectlabs/react` and we want to export a provider from `@dialectlabs/react-ui`, maybe DialectUIProvider is a better naming. What do you think?